yelp_package/extra_requirements_yelp.txt

```diff
@@ -23,18 +22,18 @@ pyopenssl==19.0.0
 PySubnetTree==0.34
 python-jsonschema-objects==0.3.1
 retrying==1.3.3
-scribereader==0.15.0
+scribereader==1.1.1
 signalform-tools==0.0.16
 slo-transcoder==3.3.0
 smmap2==2.0.3
-splunk-sdk==1.7.0
 srv-configs==1.1.0
 sticht[yelp_internal]==1.1.18
+tenacity==8.2.3
 vault-tools==0.9.2
 yelp-cgeom==1.3.1
-yelp-clog==5.2.3
+yelp-clog==7.1.1
```

Comment:
Hopefully this is fine and doesn't impact anything else already using clog? I do know there's a wiki page named "Upgrading to yelp-clog version 7+" so that might be good to take a look at if you haven't seen it already about if there's anything else to modify as part of this upgrade

Reply:
overlooked that. thanks

Reply:
opened a separate PR to upgrade clog: https://github.com/Yelp/paasta/pull/3868